package.json

```diff
@@ -9,35 +9,21 @@
   "module": "esm/history.js",
   "unpkg": "umd/history.js",
   "files": [
-    "DOMUtils.js",
-    "ExecutionEnvironment.js",
-    "LocationUtils.js",
-    "PathUtils.js",
     "cjs",
-    "createBrowserHistory.js",
-    "createHashHistory.js",
-    "createMemoryHistory.js",
-    "createTransitionManager.js",
-    "es",
     "esm",
-    "umd",
-    "warnAboutDeprecatedCJSRequire.js"
+    "umd"
   ],
   "sideEffects": false,
   "scripts": {
     "build": "rollup -c",
     "clean": "git clean -fdX .",
-    "lint": "eslint modules",
+    "lint": "eslint .",
     "prepublishOnly": "yarn build",
     "test": "karma start --single-run"
   },
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "loose-envify": "^1.2.0",
-    "resolve-pathname": "^3.0.0",
-    "tiny-invariant": "^1.0.2",
-    "tiny-warning": "^1.0.0",
-    "value-equal": "^1.0.1"
+    "loose-envify": "^1.2.0"
```

Comment:
You can probably drop this. It's for Browserify users for the `process.env` stuff. Honestly, that doesn't belong in packages; it's a bundler problem. I'm not even sure why including it in the packages themselves fixes it. But if it needs to get fixed, that should happen on Browserify's end, not on literally _every_ package in npm that uses `process.env`.